c/examples/haploid_wright_fisher.c

```diff
@@ -25,8 +27,8 @@ simulate(tsk_table_collection_t *tables, int N, int T, int simplify_interval, gs
     tables->sequence_length = 1.0;
     parents = buffer;
     for (j = 0; j < N; j++) {
-        parents[j] = tsk_node_table_add_row(&tables->nodes, 0, T,
-                TSK_NULL, TSK_NULL, NULL, 0);
+        parents[j]
```

Comment:
This looks weird to me - why is it putting the ``= tsk...()`` on the next line?

Reply:
This is due to line lenght, I think. There are params one can add to allow slop in breaking due to lenght, kinda like what black allows.